dolphinscheduler-ui/src/views/projects/task/components/node/tasks/use-hive-cli.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { reactive } from 'vue'
+import * as Fields from '../fields/index'
+import type { IJsonItem, INodeData, ITaskData } from '../types'
+
+export function useHiveCli({
+                               projectCode,
+                               from = 0,
+                               readonly,
+                               data
+                           }: {
+    projectCode: number
```

Comment:
It's better to format the code by eslint.

Reply:
May I ask whether how I could format it with `eslint`? Is there any command I could use to run?

Reply:
run 'npm run lint'

Reply:
@Amy0104 I ran `npm run lint` under `dolphinscheduler-ui` directory, thus some other files are reformatted as well. PTAL when available, thanks!

Reply:
We will add a husky to lint code later.